demos/mri_reconstruction_demo/cpp/main.cpp

```diff
@@ -38,16 +38,31 @@ int main(int argc, char** argv) {
         return 0;
     }
 
-    slog::info << *InferenceEngine::GetInferenceEngineVersion() << slog::endl;
-    InferenceEngine::Core ie;
+    slog::info << ov::get_openvino_version() << slog::endl;
+    ov::runtime::Core core;
 
-    InferenceEngine::CNNNetwork net = ie.ReadNetwork(FLAGS_m);
-    net.getInputsInfo().begin()->second->setLayout(InferenceEngine::Layout::NHWC);
+    slog::info << "Reading model " << FLAGS_m << slog::endl;
+    std::shared_ptr<ov::Model> model = core.read_model(FLAGS_m);
+    logBasicModelInfo(model);
 
-    InferenceEngine::ExecutableNetwork execNet = ie.LoadNetwork(net, FLAGS_d);
-    logExecNetworkInfo(execNet, FLAGS_m, FLAGS_d);
+    std::string outputTensorName = "";
+    for (const auto& output : model->outputs()) {
+        if (output.get_any_name().find("89") != std::string::npos) {
```

Comment:
I have checked the output names with Python API, the desired output tensor has 3 possible names: `{'StatefulPartitionedCall/model/conv2d_43/BiasAdd:0', 'Identity:0', 'Func/StatefulPartitionedCall/output/_89:0', 'StatefulPartitionedCall/Identity:0'}`  

 The `get_any_name()` might return any of them. So please, use `get_names()` method, it's safe and deterministic. Probably you should iterate through all of them to check the condition

Reply:
Maybe more stable approach is to find desired output by it's shape. In this case we need output with only 1 channel, so it would be better to find and get output with 1 channel.

Reply:
yes, also possible. would not say it is more stable, but should be good enough. This particular demo support only one specific model, so most stable approach is to check all input/output params (name, type, shape), num of inputs/outputs but it is not critically important, demo any way will not work with wrong model.

Reply:
I support the approach with the verification by the last dimension in shape, let's apply it